### PR TITLE
Drop AppVeyor

### PR DESCRIPTION
**Pull request by mattBrzezinski**

### CODE REVIEW

**Review by oxinabox — APPROVED** [review]

**Review by mjram0s — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -4,10 +4,14 @@ os:
   - linux
   - osx
```
Comment (nicoleepp) [inline .travis.yml:5]: Would windows just be untested now?

Comment (mattBrzezinski) [reply .travis.yml:5]: Yes, however we don't test Windows in any of our other packages.

I can include it here if you'd like, I just wanted to be more consistent with other packages which we maintain.

Comment (ararslan) [reply .travis.yml:5]: Makes sense to test it for open source stuff in public orgs, IMO

File: .travis.yml [hunk 2]
```diff
@@ -4,10 +4,14 @@ os:
...
 julia:
-  - 1.0
+  - 1.0  # Long Term Support
+  - 1    # Current Stable
```
Comment (ararslan) [inline .travis.yml:9]:
```suggestion
  - 1    # Current Stable
  - nightly
```

**Review by nicoleepp — APPROVED** [review]

**Pull request merged**